### PR TITLE
feat: support new Panel component

### LINKED ISSUE #51: Information panel component
## Description
Extracted from https://github.com/Altinn/altinn-studio/issues/6068. 

Apps have the need to present generic information that is not connected to a specific field. For this case, we should support an information panel component that can be used to present information in a genric manner to the end-user.

## Screenshots
image

See more on figma: https://www.figma.com/file/wnBveAG2ikUspFsQwM3GNE/Altinn-Studio-Apps?node-id=15055%3A17514
## Acceptance criteria
- Information component available in app frontend
- Can drag and drop in studio
- Json schema updated with new component

## Specification tasks

- [x] Development tasks are defined

## Development tasks

- [ ] Add component to app frontend (extend component used in #19 )
- [ ] Add to studio

## Test
- [ ] Unit
- [ ] Cypress

## Definition of done

- [ ] Documentation (docs.altinn.studio) is updated (if relevant)
  - [ ] Technical documentation
  - [ ] User documentation
- [ ] QA
- [ ] Manual test is complete (if relevant)
- [ ] Automated test is implemented (if relevant)
- [ ] All tasks in this userstory are closed (i.e. remaining tasks are moved to other user stories or marked obsolete)

### PR DESCRIPTION
**Pull request by haakemon**
## Description
- Support Panel component in formlayout

## Fixes
- fixes #51

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [x] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable) https://github.com/Altinn/altinn-studio-docs/pull/539
- [x] PR for formlayout schema update https://github.com/Altinn/altinn-cdn/pull/100
- [x] Update ui-components app with Panel example usage https://dev.altinn.studio/repos/ttd/ui-components/pulls/5
- [x] Create issue for Studio support https://github.com/Altinn/altinn-studio/issues/8599

### CODE REVIEW

File: src/altinn-app-frontend/package.json
Comment (nkylstad) [inline src/altinn-app-frontend/package.json:21]: 🥳

**Review by nkylstad — APPROVED** [review]
Looks great! 🌻

**Comment by haakemon** [comment]
Back to draft - waiting until #51 is merged

File: src/altinn-app-frontend/setupTests.ts
Comment (lasseklovstad) [inline src/altinn-app-frontend/setupTests.ts:14]: Det finnes en funksjon for dette i testUtils mockMediaQuery.

File: src/altinn-app-frontend/src/components/index.ts
Comment (nkylstad) [inline src/altinn-app-frontend/src/components/index.ts:184]: This change seems to be the opposite of the naming convention for all the other components here - is there a reason for this?

File: src/altinn-app-frontend/src/components/ConditionalWrapper.tsx
Comment (lorang92) [inline src/altinn-app-frontend/src/components/ConditionalWrapper.tsx:9]: Nice ❤️

**Review by lorang92 — APPROVED** [review]
LGTM 🥳

**Review by nkylstad — APPROVED** [review]
Looks great! 🐦 
One small comment to consider

File: src/altinn-app-frontend/src/components/index.ts
Comment (haakemon) [reply src/altinn-app-frontend/src/components/index.ts:184]: It was mainly to get rid of some [reported code smells from sonarcloud](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&id=Altinn_app-frontend-react&open=AX_0PbP8ruzTKI3lOPDv), although I think the report is showing a false positive here. It mentions that `AddressComponent` is already declared, but that shouldnt matter since its delared within the enum, so seems like it doesnt handle enums very well?

Even though the original code was not problematic after all, I though it would be good to get rid of the reported code smells just to reduce noise. The name here is not exposed anywhere as far as I can tell so it shouldnt cause any problems.

**Pull request merged**
